tests/unit/test_pywbemcli_operations.py

```diff
@@ -0,0 +1,852 @@
+# (C) Copyright 2020 IBM Corp.
+# (C) Copyright 2020 Inova Development Inc.
+# All Rights Reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Unit tests for _pywbemcli_operations module.
+"""
+
+from __future__ import absolute_import, print_function
+
+import sys
+import os
+import glob
+import re
+import pytest
+import pywbem
+
+from pywbemtools.pywbemcli._pywbemcli_operations import PYWBEMCLIFakedConnection
+from pywbemtools.pywbemcli._utils import _ensure_unicode
+from pywbemtools.pywbemcli.mockscripts import DeprecatedSetupWarning, \
+    SetupNotSupportedError
+
+from .pytest_extensions import simplified_test_function
+from .utils import captured_output
+
+OK = True
+FAIL = False
+PDB = "pdb"
+
+# Click (as of 7.1.2) raises UnsupportedOperation in click.echo() when
```

Comment:
MIght note that this is apparently fixed for click 8.0

Reply:
Who knows. Click 8.0 is not relased yet, and this goes on for quite some time now. The Click issue is referenced in the text and that allows figuring out whether and when the fix is released. NOCHANGE.